display.js

```diff
@@ -0,0 +1,52 @@
+export const getCellElem = function(cell){
+    return document.querySelector(`.row-${cell.row}-col-${cell.col}`);
+}
+
+export const displayRevealedCell = function (cell) {
+    let cellElem = getCellElem(cell);
+    let innerSpan = cellElem.querySelector("span");
+    innerSpan.classList.remove("hide");
+    innerSpan.classList.add("revealed");
+    cellElem.classList.remove("clickable");
+};
+
+
+
+export const displayGrid = function(gridObject) {
```

Comment:
not the biggest fan of the `gridObject` naming throughout the project, I'd prefer just `grid` like you have `cell`.

Reply:
otherwise this function is great, perfect size and conciseness.